src/tests/ftest/util/server_utils.py

```diff
@@ -464,7 +464,8 @@ def prepare_storage(self, user, using_dcpm=None, using_nvme=None):
             cmd.sub_command_class.sub_command_class.nvme_only.value = True
 
         if using_nvme:
-            cmd.sub_command_class.sub_command_class.hugepages.value = 4096
+            hugepages = self.manager.job.yaml.get_value("nr_hugepages")
```

Comment:
The safer method to perform the same action - https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L949:
```suggestion
            hugepages = self.get_config_value("nr_hugepages")
```